src/nacl/bindings/crypto_pwhash.py

```diff
@@ -223,3 +265,143 @@ def crypto_pwhash_scryptsalsa208sha256_str_verify(passwd_hash, passwd):
            raising=exc.InvalidkeyError)
     # all went well, therefore:
     return True
+
+
+def crypto_pwhash_argon2i(outlen, passwd, salt, opslimit, memlimit, alg):
+    """
+    Derive a raw cryptographic key using the ``passwd`` and the ``salt``
+    given as input.
+
+    :param outlen: the length of the derived key
+    :type outlen: output key length
```

Comment:
I'm away from my workstation for a couple of days; will add the needed change in a few days.